javascripts/events/eventsEvent.js

```diff
@@ -30,16 +65,32 @@ const bindEventsData = () => {
       'event': `${$('#eventName').val()}`,
       'location': `${$('#eventLocation').val()}`,
       'startDate': `${$('#eventDate').val()}`,
-      'userUid': ``,
+      'userUid': auth.getUID(),
     };
     eventsDataGateKP.requestEventPOST(eventToAdd);
     eventsDataGateKP.requestEventGET();
     $('#events-header-view').removeClass('hide');
   });
 
+  $('#events-view').on('click', '#event-btn-update', e => {
+    let eventId = $(e.target).closest('div');
+    eventId = eventId[0].id;
+    const eventToUpdate = {
+      'event': $('#eventName').val(),
+      'location': $('#eventLocation').val(),
+      'startDate': $('#eventDate').val(),
+      'userUid': auth.getUID(),
+    };
+    $('#events-header-view').removeClass('hide');
+    $('#events-header-buttons').removeClass('hide');
+    $('#events-view-nav').removeClass('hide');
+    eventsDataGateKP.requestEventPUT(eventToUpdate, eventId);
+    eventsDataGateKP.requestEventGET();
```

Comment:
remove line 88 and it works, i think its because youre calling the same function twice.  once here, and then again at the other end of your requestEventPUT function in eventDataGateKP.js